src/App.tsx

```diff
@@ -1,8 +1,6 @@
-import React, {useEffect, useState} from 'react';
```

Comment:
I think we still need `import React from 'react';` right?

Reply:
technically, no. It's an unused import

Reply:
Ah I see. That's a new feature of React 17 https://www.chakshunyu.com/blog/why-did-and-dont-you-need-to-import-react/